react/week1/todo-list-app/src/Components/Brush.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Brush = props => {
+    console.log(props)
+    return (
+        <div>
+           <li>{props.description} {":"} {props.deadline}</li> 
```

Comment:
You dont need to create component of each string item, instant create component which take array and show it. so consider creating component like Todo >  TodoList > TodoItem